repository-data/webfiles/src/main/resources/site/freemarker/common/apicatalogue/scrollable-filter-nav.ftl

```diff
@@ -0,0 +1,94 @@
+                            <label for="<#if responsive>responsive_</#if>toggler_${section.key}">
+                        <span class="icon">
+                            <span class="nhsd-a-icon nhsd-a-icon--size-xxs">
+                                <svg xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid meet" aria-hidden="true" focusable="false" viewBox="0 0 16 16" width="100%" height="100%">
+                                    <path d="M8,12L1,5.5L2.5,4L8,9.2L13.5,4L15,5.5L8,12z"></path>
+                                </svg>
+                            </span>
+                        </span>
+                                <span class="nhsd-m-filter-menu-section__heading-text nhsd-t-body-s nhsd-!t-margin-bottom-0 nhsd-!t-padding-0 <#if section.expanded>selected</#if>">
+                            ${section.displayName}
+                        </span>
+                            </label>
+                            <hr class="nhsd-a-horizontal-rule nhsd-a-horizontal-rule--size-s">
+                            <div class="section-entries">
+                                <#list section.entries as filter>
+                                    <@filterTemplate filter filtersModel 0></@filterTemplate>
+                                </#list>
+                            </div>
+                            <#if section.entries??>
+                                <hr class="nhsd-a-horizontal-rule nhsd-a-horizontal-rule--size-s">
+                            </#if>
+                        </div>
+                    </#if>
+                </#list>
+            </nav>
+        </div>
+    </#if>
+</#macro>
+
+<#macro filterTemplate filter filtersModel indentationLevel=0>
+    <#if filter.displayed>
+        <@hst.renderURL var="filterLink">
+            <#if filter.selected>
+                <@hst.param name="filters" value="${filtersModel.selectedFiltersKeysMinus(filter.key)?join(',')}" />
+            <#else>
+                <@hst.param name="filters" value="${filtersModel.selectedFiltersKeys()?join(',', '', ',')}${filter.key}" />
+            </#if>
+        </@hst.renderURL>
+        <span class="nhsd-a-checkbox">
+            <label>
+            <#if filter.selectable>
+                <a title="Filter by ${filter.displayName}" href="${filterLink}" class="nhsd-a-checkbox__label nhsd-t-body-s <#if filter.selected>selected</#if>">
+                    <input type="checkbox">${filter.displayName}
+                </a>
+                <span class="checkmark <#if filter.selected>selected</#if>"></span>
+            <#else>
+                <a href="" class="nhsd-a-checkbox__label nhsd-t-body-s nhsd-api-catalogue__filter-unavailable">
+                    <input type="checkbox" disabled>${filter.displayName}
+                </a>
+                <span class="checkmark <#if filter.selected>selected</#if>"></span>
+            </#if>
+                <p class="nhsd-a-checkbox__hint"></p>
```

Comment:
Is this element required?

Reply:
Thanks, @LEJA3 !

It was placed here replicating example from the [Design System](https://design-system.digital.nhs.uk/storybook/latest/?path=/docs/design-system-components-molecules-filter-menu-section--lab-component).

I totally agree that it's superfluous in our case and I did try removing it before submitting this PR but it broke the layout (resulting in check-boxes and their labels being displayed in a single row), so I decided to leave it in.